source/developers/tutorial.markdown

```diff
@@ -0,0 +1,140 @@
+---
+layout: page
+title: "Development tutorial"
+description: "Tutorial about the first steps on Home Assistant development"
+date: 2016-02-20 07:00
+sidebar: false
+comments: false
+sharing: true
+footer: true
+---
+
+This is a simple tutorial on how to write a component for [Home Assistant](https://home-assistant.io/). We will work on a component called "information". The purpose of this component is to display a given text in the frontend. 
+
+The setup of a development environment is described in the [Developers section](/developers/#starting-development) of the documentation.
+
+## {% linkable_title Component %}
+
+
+As a start, create a file `information.py` in your Git checkout of Home Assistant in the folder `homeassistant/component/`.
+
+```python
+"""
+homeassistant.components.information
```

Comment:
I've been trying to adhere more files to the PEP257 documentation standard. Part of this is that we should update our headers to have a description of the module in the first line. As people reading this tutorial might become contributors it would be great if we give the right example here.